src/client/src/App.js

```diff
@@ -42,7 +42,8 @@ class App extends Component {
         return (
           <div className="App">
             <header className="App-header">
-              <img src={logo} className="App-logo" alt="logo" />
+              <img src={logo} alt="logo" />
+	      <p>Welcome to ChatNow, {JSON.stringify(this.state.user.name)}</p>
```

Comment:
Is it necessary to `stringify`?

Reply:
believe so otherwise it will just show [Object object]